scss/_utility.scss

```diff
@@ -0,0 +1,37 @@
+.card-grid {
+  display: grid;
+  align-content: start;
+  gap: 20px;
+  padding: 20px 4px;
+}
+
+.card-grid > *:last-child {
+  position: relative;
+}
+.card-grid > *:last-child:after {
+  content: '';
+  display: block;
+  position: absolute;
+  top: 100%;
+  left: 0;
+  width: 100%;
+  height: 20px;
+}
+
+.center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.m-0 {
+  margin: 0;
+}
+
+.m-5 {
+  margin: 5px;
+}
+
+.scrolling {
+  overflow-y: scroll;
+}
```

Comment:
Bitte `.pos-rel: {position: relative;}` einfügen, die brauchen wir für's Bookmark in storybook